tasklist/app/views/tasks/new.html.erb

```diff
@@ -1,7 +1,5 @@
-<link href="tasklist/app/assets/stylesheets/application.css" rel="stylesheet">
-<link href='https://fonts.googleapis.com/css?family=Julius+Sans+One' rel='stylesheet' type='text/css'>
-<link href='https://fonts.googleapis.com/css?family=Megrim' rel='stylesheet' type='text/css'>
-
+<html>
```

Comment:
I think you don't need the `html` and `body` tags here, as they will already be provided by the `layouts/application.html.erb` layout file.